Content.Server/Interaction/InteractionSystem.cs

```diff
@@ -264,12 +281,29 @@ private bool HandleTryPullObject(ICommonSession? session, EntityCoordinates coor
             return pull.TogglePull(userEntity);
         }
 
-        public async void UserInteraction(IEntity user, EntityCoordinates coordinates, EntityUid clickedUid)
+        /// <summary>
+        ///     Resolves user interactions with objects.
+        /// </summary>
+        /// <remarks>
+        ///     Checks Whether combat mode is enabled and whether the user can actually interact with the given entity. 
+        /// </remarks>
+        /// <param name="altInteract">Whether to use default or alternative interactions (usually as a result of
+        /// alt+clicking). If combat mode is enabled, the alternative action is to perform the default non-combat
+        /// interaction. Having an item in the active hand also disables alternative interactions.</param>
+        public async void UserInteraction(IEntity user, EntityCoordinates coordinates, EntityUid clickedUid, bool altInteract = false )
         {
             if (user.TryGetComponent(out CombatModeComponent? combatMode) && combatMode.IsInCombatMode)
             {
-                DoAttack(user, coordinates, false, clickedUid);
-                return;
+                // alt-interact in combat mode results in normal non-combat interactions
+                // TODO COMBAT Consider using alt-interact for advanced combat? maybe alt-interact disarms?
```

Comment:
definitely should be considered for the future; tg solved it by overloading rightclick to be disarm but I really dislike it for playing since you have to do more keypresses to open the context menu